akka-docs/src/test/scala/docs/stream/cookbook/RecipeLoggingElements.scala

```diff
@@ -30,7 +30,13 @@ class RecipeLoggingElements extends RecipeSpec {
       //#log-custom
       // customise log levels
       mySource.log("before-map")
-        .withAttributes(Attributes.logLevels(onElement = Logging.WarningLevel))
+        .withAttributes(
+          Attributes.logLevels(
+            onElement = Logging.WarningLevel,
+            onFinish = Logging.InfoLevel,
+            onFailure = Logging.DebugLevel
```

Comment:
To show that users can configure different log level for each of element, finish and failure.

Reply:
👍